tests/downloads/qml/test_single_download.cpp

```diff
@@ -434,18 +434,19 @@ TestSingleDownload::testSetHashNullptr() {
     QScopedPointer<TestableSingleDownload> singleDownload(
         new TestableSingleDownload(nullptr, _man));
 
-    singleDownload->setHash("c4e5f7fcbcef75924b2abde2b2e75f3f");
+    singleDownload->setHash(QString("c4e5f7fcbcef75924b2abde2b2e75f3f"));
     // ensure that the mocks are not called and we do not crash
     verifyMocks();
 }
 
+/* TODO Fix tests
```

Comment:
Instead of adding comments like this, I think it would be better to use the [QEXPECT_FAIL](https://doc.qt.io/qt-5/qtest.html#QEXPECT_FAIL) macro, to link to the issue.